build failure on !linux !windows platforms

**Description**

https://github.com/moby/moby/pull/39292 broke compilation on !linux !windows build tags; https://github.com/moby/moby/blob/70f67c6240bbb734a9f1720ab0423b58ba957110/pkg/chrootarchive/archive_unix.go#L135 (build tag !windows) uses realChroot, defined here: https://github.com/moby/moby/blob/70f67c6240bbb734a9f1720ab0423b58ba957110/pkg/chrootarchive/chroot_linux.go#L105 (implicit build tag linux)

I'm not sure I understand the nature of the security fix well enough to fix this myself.
**Steps to reproduce the issue:**
1. go build with the build tag darwin

**Describe the results you received:**
`github.com/docker/docker/pkg/chrootarchive/archive_unix.go:135:12: undefined: realChroot`

**Describe the results you expected:**
working build :)

**Additional information you deem important (e.g. issue happens only occasionally):**
Should the call have been to chroot, not realChroot?

--- a/pkg/chrootarchive/chroot_unix.go
+++ b/pkg/chrootarchive/chroot_unix.go
@@ -10,3 +10,7 @@ func chroot(path string) error {
 	}
 	return unix.Chdir("/")
 }
+
+func realChroot(path string) error {
+	return chroot(path)
+}

